charter.md

```diff
@@ -60,7 +56,15 @@ We're seeking representation from as many members of the IRC community as possib
     </tbody>
 </table>
 
-The chair has ultimate responsibility for moving discussions forward and will make decisions in cases where consensus isn't possible, including returning proposals to the drawing board where necessary.
+
+## Governance
+
+The WG Chair is someone the IRCv3 contributors have designated as the Leader For Now of IRCv3. They're able to move discussions forward and make decisions in cases where consensus between contributors isn't possible. They can also return proposals to the drawing board where necessary.
+
+In extraordinary circumstances, the IRCv3 contributors may decide to depose the WG Chair and replace them with someone else (as we've done [in the past](https://github.com/ircv3/ircv3-specifications/issues/233)). In addition, the Chair may appoint someone else to replace them as Chair.
```

Comment:
This reads very weird. And vague. The point of charter/governance is specifically to avoid this kind of situation

Reply:
Reworded to remove this section